bridge.js

```diff
@@ -8,7 +8,7 @@ let cont = false;
 const selectors = {
   albumArt:
     '.cover-art-image',
-  trackName: 'a[data-testid="nowplaying-track-link"]',
+  trackName: 'div.now-playing-bar__left div.now-playing div._2d35c1726829c507fca5a9b5b1aae1a2-scss.ellipsis-one-line div.c319b99793755cc3bba709fe1b1fda42-scss.ellipsis-one-line div.react-contextmenu-wrapper span a',
```

Comment:
Have you tested this expanded selector on Chrome @MatejMecka

Reply:
Yes I have, it worked for me